pkg/api/auth_middleware.go

```diff
@@ -105,7 +105,10 @@ func AuthMiddleware(logger logging.Logger, swagger *openapi3.Swagger, authentica
 				return
 			}
 			if user != nil {
-				ctx := logging.AddFields(r.Context(), logging.Fields{logging.UserFieldKey: user.Username})
+				ctx := r.Context()
+				if isAdvancedAuth {
+					ctx = logging.AddFields(ctx, logging.Fields{logging.UserFieldKey: user.Username})
```

Comment:
IIUC This should stay X-LakeFS-Client is the enhanced capability that can be customized with data (`client` field in audit log) 
While this field `user` is just the authenticated lakeFS user.